.github/workflows/cli.yaml

```diff
@@ -107,34 +72,11 @@ jobs:
     pack_upload_tarballs:
       runs-on: ubuntu-latest
       if: startsWith(github.ref, 'refs/tags/@sunodo/cli@')
-      env:
-        TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
-        TURBO_TEAM: ${{ vars.TURBO_TEAM }}
       steps:
             - uses: actions/checkout@v3
```

Comment:
Do we need this if this is done at the composite action?

Reply:
Yes, we need checkout to get the contents of the repo and then refer to the composition action

Reply:
Indeed. What about the checkout inside the composite, is that needed?